lib/parzival/accounts.ex

```diff
@@ -2,12 +2,23 @@ defmodule Parzival.Accounts do
   @moduledoc """
   The Accounts context.
   """
+  use Parzival.Context
 
   import Ecto.Query, warn: false
 
   alias Parzival.Accounts.{User, UserNotifier, UserToken}
   alias Parzival.Repo
 
+  def list_users(params \\ %{}) do
```

Comment:
```suggestion
  def list_users(_opts \\ []) do
```

Reply:
I updated this file in the [b485bc0](https://github.com/joinum/parzival/pull/74/commits/b485bc0ac5e3289b554870bab058086e262dc046) commit.

Essentially I add one function that was never being called due to pattern matching problems. With this fix, this function ceases to exist, for that reason I did not commit your suggestion :pray: